tools/scrape/github.py

```diff
@@ -513,10 +521,10 @@ def _extract_bundle_repos_from_dir(dir_data, depth=0):
         # Ok, there could potentially be references to vim plugins here.
         _, file_contents = get_api_page(file_data['url'])
         contents_decoded = base64.b64decode(file_contents.get('content', ''))
-        bundles_tuple = _extract_bundle_repos_from_file(contents_decoded)
+        bundles_dict = _extract_bundle_repos_from_file(contents_decoded)
 
-        if any(bundles_tuple):
-            return bundles_tuple
+        if any(bundles_dict):
```

Comment:
Did this behavior intentionally change? If not, this should probably be `if any(bundles_dict.values()):`. This condition will always be truthy because `_extract_bundle_repos_from_file` always returns a dict containing keys.